### PR TITLE
[skip ci] Change UI

### PR DESCRIPTION
**Pull request by Lagrio**
# QNotified R ( ikka Design ) Project

## 介绍 / Description

更改界面与文本
## 更改内容 / Types of Changes

- [ ] 内核 Core
- [ ] 修复 Bugfix
- [ ] 新功能 New feature
- [ ] 增强/优化 Enhancement/optimization
- [ ] 文档 Documentation
- [x] UI界面 User interface

## 由这个PR解决或关闭的Issues / Issues Fixed or Closed by This PR

-  

## 检查列表 / Checklist

- [x] 我的代码符合本仓库的代码规范 My code follows the code style of this project
- [x] 我已经测试了这些更改，它们可以正常工作，且不会破坏任何功能(或我可以解决) I have tested the changes and verified that they work and don't break anything (as well as I can manage).
- [x] 我已合并了对后续工作无意义的commit并确认不会对后续维护造成困扰 I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

### CODE REVIEW

File: app/src/main/java/nil/nadph/qnotified/activity/ConfigV2Activity.java [hunk 1]
```diff
@@ -183,7 +179,7 @@ public void handleClickEvent(View v) {
             }
             case R.id.mainV2_help: {
                 new AlertDialog.Builder(this)
-                        .setMessage("如模块无法使用,EdXp可尝试取消优化+开启兼容模式, ROOT用户可尝试 用幸运破解器-工具箱-移除odex更改 移除QQ与本模块的优化, 太极尝试取消优化")
+                        .setMessage("如模块无法使用，EdXp可尝试取消优化+开启兼容模式  ROOT用户可尝试 用幸运破解器-工具箱-移除odex更改 移除QQ与本模块的优化, 太极尝试取消优化")
```
Comment (NextAlone) [inline app/src/main/java/nil/nadph/qnotified/activity/ConfigV2Activity.java:182]:
```suggestion
                        .setMessage("如模块无法使用，EdXp可尝试取消优化+开启兼容模式  ROOT用户可尝试用 幸运破解器-工具箱-移除odex更改 移除QQ与本模块的优化, 太极尝试取消优化")
```

**Review by NextAlone — APPROVED** [review]

**Comment by singleNeuron** [comment]
你commit到我的分支啊，把我的分支关了自己开一个干啥

File: app/src/main/res/values-v23/styles.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (singleNeuron) [inline app/src/main/res/values-v23/styles.xml:1]: https://github.com/ferredoxin/QNotified/pull/405/commits/24cb0a543771c014093596a8300811dec1fd4740

这样修，别单独建个v23

**Review by singleNeuron — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-v23/styles.xml (hunk 2 above)
Comment (Lagrio) [reply app/src/main/res/values-v23/styles.xml:1]: 哦

**Review by singleNeuron — APPROVED** [review]

**Review by cinit — APPROVED** [review]

**Pull request merged**